aii-opennebula/src/test/perl/rpcdata.pm

```diff
@@ -290,7 +290,7 @@ EOF
+<IMAGE_POOL><IMAGE><ID>43</ID><UID>0</UID><GID>0</GID><UNAME>oneadmin</UNAME><GNAME>oneadmin</GNAME><NAME>node630.cubone.os_vda</NAME><PERMISSIONS><OWNER_U>1</OWNER_U><OWNER_M>1</OWNER_M><OWNER_A>0</OWNER_A><GROUP_U>0</GROUP_U><GROUP_M>0</GROUP_M><GROUP_A>0</GROUP_A><OTHER_U>0</OTHER_U><OTHER_M>0</OTHER_M><OTHER_A>0</OTHER_A></PERMISSIONS><TYPE>0</TYPE><DISK_TYPE>3</DISK_TYPE><PERSISTENT>0</PERSISTENT><REGTIME>1410854276</REGTIME><SOURCE><![CDATA[one/one-43]]></SOURCE><PATH><![CDATA[http://marketplace.c12g.com/appliance/4fc76a938fb81d3517000003/download/0]]></PATH><FSTYPE><![CDATA[]]></FSTYPE><SIZE>40</SIZE><STATE>1</STATE><RUNNING_VMS>1</RUNNING_VMS><CLONING_OPS>0</CLONING_OPS><CLONING_ID>-1</CLONING_ID><DATASTORE_ID>105</DATASTORE_ID><DATASTORE>ceph.altaria</DATASTORE><VMS><ID>60</ID></VMS><CLONES/><TEMPLATE><DEV_PREFIX><![CDATA[vd]]></DEV_PREFIX><FROM_APP><![CDATA[4fc76a938fb81d3517000003]]></FROM_APP><FROM_APP_FILE><![CDATA[0]]></FROM_APP_FILE><FROM_APP_NAME><![CDATA[ttylinux-kvm]]></FROM_APP_NAME><MD5><![CDATA[04c7d00e88fa66d9aaa34d9cf8ad6aaa]]></MD5><QUATTOR><![CDATA[1]]></QUATTOR></TEMPLATE></IMAGE></IMAGE_POOL>
```

Comment:
You are right, this xml crap is awful :smile: but it's the only way to mock the rpc endpoint